File: 02-eda-1/pie.r
df <- read.csv("movies.csv")
df <- na.omit(df)
# Reduce data to 10 records for better plot
df <- df[1:10, ]
# Convert VOTES column from string to int
df$VOTES <- as.integer(gsub(",", "", df$VOTES))

pie(df$VOTES, labels = df$MOVIES, main = "Votes")

File: 02-eda-1/pie.py
import pandas as pd
import matplotlib.pyplot as plt

df = pd.read_csv("movies.csv")
df = df.dropna()
# Reduce data to 10 records for better plot
df = df[:10]
print(df)
# Map string to int
df["VOTES"] = df["VOTES"].str.replace(",", "").astype(int)

plt.pie(df["VOTES"], labels=df["MOVIES"], autopct="%1.1f%%")
plt.title("Votes")
plt.axis("equal")
plt.show()

